.github/pull_request_template.md

```diff
@@ -0,0 +1,31 @@
+<!--
+Use this template as a guide. Omit sections that don't apply.
+More details about the template are at https://github.com/edx/open-edx-proposals/pull/180
+-->
+
+## Description
+
+Describe what this pull request changes, and why. Include implications for people using this change.
+Design decisions should be documented in the repo (docstring / README / ADR) and linked here.
+
+Useful information:
+- Which edX user roles will this change impact? Common user roles are "Learner", "Course Author",
+"Developer" and "Admin".
```

Comment:
What does "Admin" mean in this context? "Course Admin" and "Django Admin" are very different personas. Maybe include "Operator"?

Also, do we want this to be a structured field that we can parse for and use in automation? Or is this a reminder for folks to include in their description?

Finally, in order to "sell" and "maintain" this info, it would be great to know _why_ we believe this info is necessary to ask people to specify.

Reply:
I've changed it to "Operator".

As I've said, I could use some help expressing the "why" for each piece of information. I would rather not put all the whys into the PR template: they will make it bulkier and harder to navigate.  We have a draft of a document for the reasons and backstory for the information we need from developers: https://github.com/edx/open-edx-proposals/pull/180  Can we add to that document?

Reply:
Indeed. Scratch my earlier comment about putting the rationale in an ADR.